Update create-detection-rule.md (#2854)
elastic/docs-content: solutions/security/detect-and-alert/create-detection-rule.md

        You can also leave the **Group by** field undefined. The rule then creates an alert when the number of search results is equal to or greater than the threshold value. If you set **Count** to limit the results by `process.name` >= 2, an alert will only be generated for source/destination IP pairs that appear with at least 2 unique process names across all events.

        ::::{important}

>         Alerts created by threshold rules are synthetic alerts that do not resemble the source documents. The alert itself only contains data about the fields that were aggregated over (the **Group by** fields). Other fields are omitted, because they can vary across all source documents that were counted toward the threshold. Because of this, it isn’t possible to iterate on the search hits, so those aren’t available in the context. You can reference the actual count of documents that exceeded the threshold from the `kibana.alert.threshold_result.count` field. `context.alerts.kibana.alert.threshold_result.terms` contains fields and values from any group by specified by the rule. For example:

Reviewer: we do not use terminology `hits` anywhere on this page for any of alert types.
Should we just focus on available fields from original documents in resulting alert?

Reviewer 2: What would be a better way to say this?

Reviewer: Hits are results returned from ES search request. Hit == document in ES index.
For other rule types, we copy all existing fields in that document into alert.
For threshold rule, only **Group by** fields are getting copied.

In what context `hits` are referred here? Particularly `Because of this, it isn’t possible to iterate on the search hits, so those aren’t available in the context.`

Reviewer 2: ```suggestion
        Alerts created by threshold rules are synthetic alerts that do not resemble the source documents:
        
          - The alert itself only contains data about the fields that were aggregated over (the **Group by** fields specified in the rule).
          - All other fields are omitted and aren't available in the alert. This is because these fields can vary across all source documents that were counted toward the threshold. 
          - You can reference the actual count of documents that exceeded the threshold from the `kibana.alert.threshold_result.count` field. 
          - `context.alerts.kibana.alert.threshold_result.terms` contains fields and values from any **Group by** fields specified in the rule. For example:
```
How about we structure the note a little bit with something like this (because it's getting too big and hard to read). That way, the important information is easier to read and understand.
Lmk how this looks to you

Reviewer: @florent-leborgne 

I like this update

Reviewer 2: I'm committing this change, please take a look at the updated preview and let me know how this now looks to you both @vitaliidm @bczifra
